### PR TITLE
feat!: Only Project Manager and administror able to view Costing and Margin section in Project

### PR DESCRIPTION
**Pull request by vishdha**
Costing and Billing section and Margin section info visible to user with project Manager role only

### CODE REVIEW

**Comment by barredterra** [comment]
@vishdha did you test this? It doesn't seem to work for me. I tried the following:

1. Create a new site with erpnext
2. Create a new **Project** as system manager
3. Create a new **User** with role “Project User” (nothing else)
4. Log in as project user

    -> Can still see section costing & billing and margin.

(I checked the Project DocType, it was synced correctly.)

**Comment by ankush** [comment]
> Can still see section costing & billing and margin.



Should we be adding role-based perm levels in the core? 

This is specific to individual organizations and should ideally be configured during the implementation phase 😅

**Comment by barredterra** [comment]
@ankush I agree, but in this case we thought that the requirement is common enough. Frappe is probably one of the few organizations where everyone is allowed to see the financial details of a project. In my experience, most companies want to hide them from project users.

IMO it makes sense that financial details get a different permlevel than other project fields.

File: erpnext/patches/v14_0/update_project_permission.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import frappe
+from frappe.permissions import add_permission, update_permission_property
+from frappe.core.page.permission_manager.permission_manager import remove
+
```
Comment (barredterra) [inline erpnext/patches/v14_0/update_project_permission.py:4]:
```suggestion


```

**Comment by vishdha** [comment]
Closing this as customized PR added

**Comment by barredterra** [comment]
Let's get this merged anywas, if Frappe agrees

**Pull request closed without merging**